crates/telio-starcast/src/transport.rs

```diff
@@ -351,23 +394,49 @@ mod tests {
         task: Task<State>,
         transport_socket: Arc<UdpSocket>,
         channel: Chan<Vec<u8>>,
-        peers: Vec<(PublicKey, UdpSocket)>,
+        peers: Vec<(PublicKey, UdpSocket, bool, bool)>,
     }
 
     impl Scaffold {
         async fn start() -> Self {
             let transport_socket = Arc::new(Self::bind_local_socket().await);
-            let mut peers = Vec::with_capacity(3);
-            for _ in 0..3 {
-                peers.push((SecretKey::gen().public(), Self::bind_local_socket().await));
-            }
+            // Peers with all the different possible meshnet configurations:
```

Comment:
This code be made concise for eg wtih
```
for i in 0..3 {
    let allow_multicast = i % 2 == 0;  // true for i = 0, 2; false for i = 1, 3
    let peer_allow_multicast = i < 2;      // true for i = 0, 1; false for i = 2, 3

    peers.push((public_key, socket, is_first_true, is_second_true));
}
```

Reply:
I think your point is valid, but the way it is now is easier to read and conveys intention a bit better, so I think these are both valid approaches, so I'd rather not change the current code, unless there's somehting wrong with it.